sources/europe/gb/OSMUK-Cadastral-Parcels.geojson

```diff
@@ -0,0 +1,1119 @@
+{
+  "type":"Feature",
+  "properties":{
+    "name":"OSMUK Cadastral Parcels",
+    "i18n":true,
+    "type":"tms",
+    "category":"other",
+    "url":"https://tiles.osmuk.org/PropertyBoundaries/{zoom}/{x}/{y}.png",
+    "min_zoom":18,
+    "max_zoom":20,
+    "permission_osm":"implicit",
+    "license_url":"https://www.nationalarchives.gov.uk/doc/open-government-licence/version/3/",
+    "id":"OSMUK-Cadastral-Parcels",
+    "description":"Perectly aligned cadastral (land registry) parcels in Great Britain",
+    "country_code":"GB",
+    "overlay":true,
+    "attribution":{
+      "text":"Contains public sector information licensed under the Open Government Licence v3.0.",
+      "url":"https://www.openrailwaymap.org/"
```

Comment:
Is the openrailwaymap.org intended as url?

Reply:
Oh. I was so careful as well and I still managed to miss something. You can tell which overlay layer I used as the starting point! Now fixed.

Reply:
No worries!